packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -238,6 +238,46 @@ class PlatformViewsService {
     }
     return UiKitViewController._(id, layoutDirection);
   }
+
+  /// `id` is an unused unique identifier generated with [platformViewsRegistry].
+  ///
+  /// `viewType` is the identifier of the iOS view type to be created, a
+  /// factory for this view type must have been registered on the platform side.
+  /// Platform view factories are typically registered by plugin code.
+  ///
+  /// `onFocus` is a callback that will be invoked when the UIKit view asks to
+  /// get the input focus.
+  /// The `id, `viewType, and `layoutDirection` parameters must not be null.
+  /// If `creationParams` is non null then `creationParamsCodec` must not be null.
+  static Future<AppKitViewController> initAppKitView({
+    required int id,
+    required String viewType,
+    required TextDirection layoutDirection,
+    dynamic creationParams,
+    MessageCodec<dynamic>? creationParamsCodec,
+    VoidCallback? onFocus,
+  }) async {
+    assert(creationParams == null || creationParamsCodec != null);
+
+    // TODO(amirh): pass layoutDirection once the system channel supports it.
```

Comment:
This should be tied to a GitHub issue instead.

Reply:
This TODO was already present in the codebase, it has now just been copied into this new location where it is presumably also applicable. Should we create a new issue for this assuming one does not already exist?

Reply:
Yep - assuming no such issue exists we should create one. We used to add them as they're written here, but Amir hasn't worked on Flutter in years and we've updated the style guide to require a TODO in the meantime. Probably worth replacing the iOS equivalent comment too while you're in there.